flutter_libserialport does not work on Archlinux anymore

I have been usinig it for few weeks with no problem, but today it end up with these errors:

	/home/kozak/Vyvoj/Eurosat/etrackermanagetool/linux/flutter/ephemeral/.plugin_symlinks/flutter_libserialport/third_party/libserialport/linux_termios.c:98:9: error: invalid application of 'sizeof' to an incomplete type 'struct termiox'
	/home/kozak/Vyvoj/Eurosat/etrackermanagetool/linux/flutter/ephemeral/.plugin_symlinks/flutter_libserialport/third_party/libserialport/linux_termios.c:106:15: error: incomplete definition of type 'struct termiox'
	/home/kozak/Vyvoj/Eurosat/etrackermanagetool/linux/flutter/ephemeral/.plugin_symlinks/flutter_libserialport/third_party/libserialport/linux_termios.c:106:27: error: use of undeclared identifier 'RTSXOFF'
	/home/kozak/Vyvoj/Eurosat/etrackermanagetool/linux/flutter/ephemeral/.plugin_symlinks/flutter_libserialport/third_party/libserialport/linux_termios.c:107:15: error: incomplete definition of type 'struct termiox'
	/home/kozak/Vyvoj/Eurosat/etrackermanagetool/linux/flutter/ephemeral/.plugin_symlinks/flutter_libserialport/third_party/libserialport/linux_termios.c:107:27: error: use of undeclared identifier 'CTSXON'
	/home/kozak/Vyvoj/Eurosat/etrackermanagetool/linux/flutter/ephemeral/.plugin_symlinks/flutter_libserialport/third_party/libserialport/linux_termios.c:108:15: error: incomplete definition of type 'struct termiox'
	/home/kozak/Vyvoj/Eurosat/etrackermanagetool/linux/flutter/ephemeral/.plugin_symlinks/flutter_libserialport/third_party/libserialport/linux_termios.c:108:27: error: use of undeclared identifier 'DTRXOFF'
	/home/kozak/Vyvoj/Eurosat/etrackermanagetool/linux/flutter/ephemeral/.plugin_symlinks/flutter_libserialport/third_party/libserialport/linux_termios.c:109:15: error: incomplete definition of type 'struct termiox'
	/home/kozak/Vyvoj/Eurosat/etrackermanagetool/linux/flutter/ephemeral/.plugin_symlinks/flutter_libserialport/third_party/libserialport/linux_termios.c:109:27: error: use of undeclared identifier 'DSRXON'
	/home/kozak/Vyvoj/Eurosat/etrackermanagetool/linux/flutter/ephemeral/.plugin_symlinks/flutter_libserialport/third_party/libserialport/linux_termios.c:118:7: error: incomplete definition of type 'struct termiox'
	/home/kozak/Vyvoj/Eurosat/etrackermanagetool/linux/flutter/ephemeral/.plugin_symlinks/flutter_libserialport/third_party/libserialport/linux_termios.c:118:22: error: use of undeclared identifier 'RTSXOFF'
	/home/kozak/Vyvoj/Eurosat/etrackermanagetool/linux/flutter/ephemeral/.plugin_symlinks/flutter_libserialport/third_party/libserialport/linux_termios.c:118:32: error: use of undeclared identifier 'CTSXON'
	/home/kozak/Vyvoj/Eurosat/etrackermanagetool/linux/flutter/ephemeral/.plugin_symlinks/flutter_libserialport/third_party/libserialport/linux_termios.c:118:41: error: use of undeclared identifier 'DTRXOFF'
	/home/kozak/Vyvoj/Eurosat/etrackermanagetool/linux/flutter/ephemeral/.plugin_symlinks/flutter_libserialport/third_party/libserialport/linux_termios.c:118:51: error: use of undeclared identifier 'DSRXON'
	/home/kozak/Vyvoj/Eurosat/etrackermanagetool/linux/flutter/ephemeral/.plugin_symlinks/flutter_libserialport/third_party/libserialport/linux_termios.c:121:8: error: incomplete definition of type 'struct termiox'
	/home/kozak/Vyvoj/Eurosat/etrackermanagetool/linux/flutter/ephemeral/.plugin_symlinks/flutter_libserialport/third_party/libserialport/linux_termios.c:121:21: error: use of undeclared identifier 'RTSXOFF'
	/home/kozak/Vyvoj/Eurosat/etrackermanagetool/linux/flutter/ephemeral/.plugin_symlinks/flutter_libserialport/third_party/libserialport/linux_termios.c:123:8: error: incomplete definition of type 'struct termiox'
	/home/kozak/Vyvoj/Eurosat/etrackermanagetool/linux/flutter/ephemeral/.plugin_symlinks/flutter_libserialport/third_party/libserialport/linux_termios.c:123:21: error: use of undeclared identifier 'CTSXON'
	/home/kozak/Vyvoj/Eurosat/etrackermanagetool/linux/flutter/ephemeral/.plugin_symlinks/flutter_libserialport/third_party/libserialport/linux_termios.c:125:8: error: incomplete definition of type 'struct termiox'
	fatal error: too many errors emitted, stopping now [-ferror-limit=]
	Exception: Build process failed

--- a/linux/libserialport/config.h
+++ b/linux/libserialport/config.h
@@ -54,7 +54,7 @@
 /* #undef HAVE_STRUCT_TERMIOS_C_OSPEED */
 
 /* Define to 1 if the system has the type `struct termiox'. */
-#define HAVE_STRUCT_TERMIOX 1
+/* #undef HAVE_STRUCT_TERMIOX */
 
 /* sys/file.h is available. */
 #define HAVE_SYS_FILE_H 1

